modular_zubbers/code/modules/storyteller/event_defines/roleset/_antagonist_event.dm

```diff
@@ -0,0 +1,150 @@
+/datum/round_event_control/antagonist
+	reoccurence_penalty_multiplier = 0
+	track = EVENT_TRACK_ROLESET
+	/// Protected roles from the antag roll. People will not get those roles if a config is enabled
+	var/protected_roles = list(
+		JOB_CAPTAIN,
+		JOB_BLUESHIELD,
+
+		// Heads of staff
+		JOB_HEAD_OF_PERSONNEL,
+		JOB_HEAD_OF_SECURITY,
+		JOB_CHIEF_ENGINEER,
+		JOB_CHIEF_MEDICAL_OFFICER,
+		JOB_RESEARCH_DIRECTOR,
+		JOB_QUARTERMASTER,
+
+		// Seccies
+		JOB_DETECTIVE,
+		JOB_SECURITY_OFFICER,
+		JOB_WARDEN,
+		JOB_CORRECTIONS_OFFICER,
+		JOB_PRISONER,
```

Comment:
JOB_SECURITY_MEDIC,

Reply:
Sorry-To clarify, this needs to be added to the security blacklist, as secmedics are spawning as antags currently, while they need to be protected from antag roles.

Reply:
Guess who forgor.